src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -1071,6 +1066,9 @@ enum AppStrings {
 	}
 
 	enum AppInformation {
+		static let appInformationNavigationTitle = NSLocalizedString("Home_AppInformationNavigation_Title", comment: "")
+		static let appInformationVersion = NSLocalizedString("Home_AppInformationCard_Version", comment: "")
```

Comment:
```suggestion
		static let appInformationNavigationTitle = NSLocalizedString("App_Information_Title", comment: "")
		static let appInformationVersion = NSLocalizedString("App_Information_Version", comment: "")
```

That way the naming would be aligned with the other ones on the same screen. And please move the second one (the renamed `Home_AppInformationCard_Version`) to the same `/* App Information */` section in the Localizable.strings file as it is not used on the Home Screen.